infrastructure-examples/gcp_vm.yml

```diff
@@ -0,0 +1,28 @@
+cluster_name: gcloud-gke
+gcloud:
+  ssh_user: rocky
+  operating_system:
+    name: rocky-linux-8
+  regions:
+    us-west2:
+      cidr_block: 10.2.0.0/16
+      azs:
+        us-west2-b: 10.2.20.0/24
+      service_ports:
+        - port: 22
+          protocol: tcp
+          description: "SSH"
+  gke:
+    mydb2:
+      region: us-west2
+      az: us-west2-b
+      instance_type: "n1-standard-1"
+      cpu_count: 2
+      password: "12Password!"
+      settings:
+        - name: max_connections
+          value: 1000
+        - name: random_page_cost
+          value: 1.25
+        - name: work_mem
+          value: 16000
```

Comment:
`settings` does not seem to be used. Should be removed.

Reply:
Removed